build.gradle.kts

```diff
@@ -377,15 +377,21 @@ tasks {
 
   publishPlugin {
     token.set(System.getenv("PUBLISH_TOKEN"))
+
     // pluginVersion is based on the SemVer (https://semver.org) and supports pre-release labels,
     // like 2.1.7-nightly
     // Specify pre-release label to publish the plugin in a custom Release Channel automatically.
     // Read more:
     // https://plugins.jetbrains.com/docs/intellij/deployment.html#specifying-a-release-channel
-
-    channels.set(
-        listOf(
-            properties("pluginVersion").split('-').getOrElse(1) { "default" }.split('.').first()))
```

Comment:
```
.split('.').first()
```

This ☝️ was redundant. Historically, the plugin supported versions like `2.1.7-alpha.3`. It's no longer a case for us.